### PR TITLE
docs: amending pr template to show required tags

### PR DESCRIPTION
**Pull request by JasonBellONS**
# Description

Rule changes added to require one of the semantic versioning tags in order to complete a pr, amendments to the PR template to remind users of these tags

If you have not checked any of the lists below explain why here.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Technical Enhancements

# Checklist:

If any of these are not completed, please explain why in the notes.

## **Definition of Done**
**Code and merges**

- [x] Code to be commented on where applicable 
- [x] Documentation updated where required 
- [x] Have considered non-functional requirements such as Security, Performance, Scalability, and Fault Tolerance
- [x] I have linted the code

**Testing**

- [x] All levels of acceptance test are passing (automated, integration, manual, accessibility, etc.)
- [x] I have run the behave command to check the selenium behaviour tests pass locally
- [x] Acceptance criteria met

**Other Checks**
- [x] I have performed a self-review of my own code
- [x] I have checked for spelling errors
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes don't break anything unexpected
- [x] I have checked and updated the security.txt file where required
- [x] Up to date with the main branch
- [x] I have a commit with the fix:, feat or BREAKING_CHANGE: tags and will copy this commit message to the squash
commit message

### CODE REVIEW

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://d3uyg76goah86z.cloudfront.net/
cloudfront_id=E29DBI82J6YN59

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://d3uyg76goah86z.cloudfront.net/
cloudfront_id=E29DBI82J6YN59

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -38,5 +38,8 @@ If any of these are not completed, please explain why in the notes.
 - [ ] My changes don't break anything unexpected
 - [ ] I have checked and updated the security.txt file where required
 - [ ] Up to date with the main branch
-- [ ] I have a commit with the fix:, feat or BREAKING_CHANGE: tags and will copy this commit message to the squash
+- [ ] I have a commit with the fix:, feat or BREAKING_CHANGE: tags (or other relevant tag) and will copy this commit message to the squash
 commit message
+
+Note when merging, squash commit must begin with one of the following tags:
```
Comment (gibbardsteve) [inline .github/pull_request_template.md:44]: Could you add examples of the format and use cases?  When would we use "chore" (if we don't expect to use it don't mention in the list).  

E.g 
**fix:** - use when you are fixing an issue that has been found in the existing solution
example commit message - _fix: Missing link on Help Centre page_

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://d3uyg76goah86z.cloudfront.net/
cloudfront_id=E29DBI82J6YN59

**Review by gibbardsteve — APPROVED** [review]

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://d3uyg76goah86z.cloudfront.net/
cloudfront_id=E29DBI82J6YN59

**Pull request merged**